### DISCUSSION TITLE
account is suspended

### DISCUSSION THREAD

**Discussion by ghost** [community/community#146229] [Discussions]
### Select Topic Area

Question

### Body

Hello GitHub staff, thank you for providing such a great platform. I am also an open source enthusiast. I have been contributing code to many communities, but I don't know why you banned me. I have already had feelings for this, and those contributions and permissions are very meaningful to me. Please tell me the reason, I will deal with any illegal operations, and I hope you can restore my account.
https://support.github.com/ticket/personal/0/3125234

GitHub is very important to me. I have made many contributions to open source and I want to continue to participate in open source projects.
I realize my mistakes and will be very cautious when forking projects in the future.
My own projects do not violate the DMCA, and there are other people who participate in these open source projects. It is unfair to other contributors to directly block me and these projects.
It is tough to say that virtual assets are also assets and need to be protected.

https://github.com/thinkasany
![image](https://github.com/user-attachments/assets/7315616e-5a83-4d55-8ce4-994e9b41947b)
![image](https://github.com/user-attachments/assets/b9695e17-2db3-4475-ad73-4fe6480f2b04)

**Comment by mecodeatlas** [comment]
Hi @ThinkerWing

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

**Reply by ghost** [reply]
@mecodeatlas Hi, thank you for your recovery, but I suspect there is a blocking bug in github's discussions. Three of my accounts have been blocked at https://github.com/ant-design/ant-design/discussions, namely thinkasany, thinkasany2, and nanyizhixin163. All of them were editing draft requests for comments with legal and compliant content, but I don't know why they were blocked. Is there any robot-related strategy or rule? But I did operate it manually. I want to understand why such a problem occurred. It has a great impact on my work and life. I have contributed to github for many years and have been active in many large repositories, including many of my own personal projects. I am really sad and hope you can help me.
